it/spring/boot2-tomcat9/src/test/java/com/linecorp/armeria/spring/tomcat/demo/ErrorHandlingTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.spring.tomcat.demo;
+
+import static net.javacrumbs.jsonunit.fluent.JsonFluentAssert.assertThatJson;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.linecorp.armeria.spring.LocalArmeriaPort;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+public class ErrorHandlingTest {
+
+    @LocalArmeriaPort
+    private int port;
+
+    @Inject
+    private TestRestTemplate restTemplate;
+
+    private static String baseUrl(int port) {
+        return "http://localhost:" + port + "/tomcat/api/rest/v1";
+    }
+
+    @Test
+    public void runtimeExceptionShouldReturnSpringFormattedMessage() throws Exception {
+        final ResponseEntity<String> response =
+                restTemplate.getForEntity(baseUrl(port) + "/error-handling/runtime-exception", String.class);
```

Comment:
From the client's point of view, the full path is `/tomcat/api/rest/v1/error-handling/runtime-exception`, but Tomcat has no way to know this, so it gives the following response. (There seems to be no way to fix this.)

```json
{"timestamp":"2021-08-02T06:02:51.573+00:00","status":500,"error":"Internal Server Error","message":"runtime exception","path":"/error-handling/runtime-exception"}
```

what do you think? Should we ignore this as a known issue?

Reply:
Yeah, let's keep it as a known issue for now.

Reply:
Thanks for feedback 👍